tests/UI/draggableTests.spec.js

```diff
@@ -0,0 +1,36 @@
+import { test } from '../../modules/base';
```

Comment:
Sredi malo naming test case-ova, idealno bi bilo da sadrzi expected result u opisu, npr. "Should be able to drag across X axis/Shouldn't be able to...".